### PR TITLE
Add react application with Contacts List page

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: packages/react/src/contexts/navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <NavigationContext.Provider
+      value={{ navigationData, setNavigationData }}
+      {...props}
+    />
+  );
+}
+
+function withNavigationWatcher(Component: React.ElementType, path: string) {
+  const WrappedComponent = function (props: any) { //unknown check with Platforms
```
Comment (Krijovnick) [inline packages/react/src/contexts/navigation.tsx:19]: props: Record<string, unknown>

File: packages/react/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+root.render(
+  // <React.StrictMode> // T1094699
```
Comment (Krijovnick) [inline packages/react/src/index.tsx:12]: The bug already fixed. Maybe this block of code can be uncommented?

File: packages/react/src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export { default as CRMContactList } from './crm-contact-list/crm-contact-list';
+
+// next pages
```
Comment (Krijovnick) [inline packages/react/src/pages/index.tsx:3]: Are commented blocks of code necessary here?

File: packages/react/src/index.tsx (hunk 2 above)
Comment (groshenkovamarina) [reply packages/react/src/index.tsx:12]: I checked the fix hasn't yet been included in any of the current release versions

File: packages/react/src/contexts/navigation.tsx (hunk 1 above)
Comment (groshenkovamarina) [reply packages/react/src/contexts/navigation.tsx]: Fixed

File: packages/react/src/app-routes.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { 
+  // CRMContactForm, 
+  CRMContactList, 
+  // PlanningTaskList, 
+  // PlanningTaskDetails 
+} from './pages';
+import { withNavigationWatcher } from './contexts/navigation';
+
+const routes = [
+    // {
```
[the remark this answers is not in the document]
Comment (groshenkovamarina) [reply packages/react/src/app-routes.tsx]: I removed comments and added empty future pages

File: packages/react/src/pages/index.tsx (hunk 3 above)
Comment (groshenkovamarina) [reply packages/react/src/pages/index.tsx]: I removed comments and added empty future pages

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**